src/test/java/org/javarosa/test_utils/ReflectionUtils.java

```diff
@@ -0,0 +1,62 @@
+package org.javarosa.test_utils;
```

Comment:
you can probaby just do `org.javarosa.utils` as a package.

Reply:
Since that package already exists I think it makes sense to use it

Reply:
ahh didn't realise it already exists. makes sense.